From 025cefd9aa46eaa28ae3c3f1699ea48bcee599dd Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 4 May 2015 12:18:37 -0500
Subject: [PATCH] bb.org defaults

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 settings/standalone.js | 12 ++++++++----
 1 file changed, 8 insertions(+), 4 deletions(-)

diff --git a/settings/standalone.js b/settings/standalone.js
index 4fd9ac7..7cb0f75 100644
--- a/settings/standalone.js
+++ b/settings/standalone.js
@@ -30,7 +30,8 @@ module.exports = function(manifest, installPath) {
         standalone: true,
         startBridge: true,
         manifest: manifest,
-        workspaceDir: workspaceDir,
+        //workspaceDir: workspaceDir,
+        workspaceDir: "/var/lib/cloud9",
         projectName: path.basename(workspaceDir),
         homeDir: home,
         workspaceId: "devel",
@@ -41,12 +42,14 @@ module.exports = function(manifest, installPath) {
         dev: true,
         sdk: sdk,
         pid: process.pid,
-        port: process.env.PORT || 8181,
+        //port: process.env.PORT || 8181,
+        port: process.env.PORT || 3000,
         host: process.env.IP || "0.0.0.0",
         testing: false,
         platform: process.platform,
         arch: process.arch,
-        tmux: path.join(installPath, "bin/tmux"),
+        //tmux: path.join(installPath, "bin/tmux"),
+        tmux: "/usr/bin/tmux",
         nakBin: path.join(__dirname, "../node_modules/nak/bin/nak"),
         bashBin: "bash",
         nodeBin: [process.execPath],
@@ -61,7 +64,8 @@ module.exports = function(manifest, installPath) {
         homeUrl: "/home",
         collab: false,
         installed: true,
-        packed: false,
+        //packed: false,
+        packed: true,
         packedThemes: true,
         readonly: false,
         role: "a",
-- 
2.1.4

