From 18a6bcd88d9d47ea68dfafe52f59319b0b306c9f Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 24 Feb 2015 12:50:36 -0600
Subject: [PATCH] bb.org defaults

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 settings/standalone.js | 12 ++++++++----
 1 file changed, 8 insertions(+), 4 deletions(-)

diff --git a/settings/standalone.js b/settings/standalone.js
index 92a162c..1f8d03e 100644
--- a/settings/standalone.js
+++ b/settings/standalone.js
@@ -28,7 +28,8 @@ module.exports = function(manifest, installPath) {
     var config = {
         standalone: true,
         manifest: manifest,
-        workspaceDir: workspaceDir,
+        //workspaceDir: workspaceDir,
+        workspaceDir: "/var/lib/cloud9",
         projectName: path.basename(workspaceDir),
         homeDir: home,
         workspaceId: "devel",
@@ -38,11 +39,13 @@ module.exports = function(manifest, installPath) {
         uid: "-1",
         dev: true,
         pid: process.pid,
-        port: process.env.PORT || 8181,
+        //port: process.env.PORT || 8181,
+        port: process.env.PORT || 3000,
         host: process.env.IP || "0.0.0.0",
         testing: false,
         platform: process.platform,
-        tmux: path.join(installPath, "bin/tmux"),
+        //tmux: path.join(installPath, "bin/tmux"),
+        tmux: "/usr/bin/tmux",
         nakBin: path.join(__dirname, "../node_modules/nak/bin/nak"),
         bashBin: "bash",
         nodeBin: [process.execPath],
@@ -57,7 +60,8 @@ module.exports = function(manifest, installPath) {
         homeUrl: "/home",
         collab: false,
         installed: true,
-        packed: false,
+        //packed: false,
+        packed: true,
         packedThemes: true,
         readonly: false,
         role: "a",
-- 
2.1.4

