From 3bcd9d2b55f79aafb03185c4169eded5dd373c75 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 21 Sep 2015 09:56:37 -0500
Subject: [PATCH 3/3] core: dont updateCore (we checkout a sha commit) and dont
 autorun

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 scripts/install-sdk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/install-sdk.sh b/scripts/install-sdk.sh
index 70f0eb1..4b9868b 100755
--- a/scripts/install-sdk.sh
+++ b/scripts/install-sdk.sh
@@ -157,7 +157,7 @@ NODE=node
 rm -rf ./build/standalone
 
 # pull the latest version
-updateCore || true
+#updateCore || true
 
 installGlobalDeps
 updateAllPackages
-- 
2.7.0.rc3

