From 5edeee5f1df449e33377356bccaaf524117bb9c1 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 4 May 2015 12:22:02 -0500
Subject: [PATCH 3/3] core: dont updateCore (we checkout a sha commit) and dont
 autorun

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 scripts/install-sdk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/install-sdk.sh b/scripts/install-sdk.sh
index 2a7fda8..b1042bd 100755
--- a/scripts/install-sdk.sh
+++ b/scripts/install-sdk.sh
@@ -128,7 +128,7 @@ NODE=node
 rm -rf ./build/standalone
 
 # pull the latest version
-updateCore || true
+#updateCore || true
 
 installGlobalDeps
 updateAllPackages
-- 
2.1.4

