From 51c21990bf83e3ffb111187ab9b9eeb6c587bb99 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 11 Mar 2015 16:21:41 -0500
Subject: [PATCH 3/3] core: dont updateCore (we checkout a sha commit) and dont
 autorun

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 scripts/install-sdk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/install-sdk.sh b/scripts/install-sdk.sh
index 2ce6fe1..4841775 100755
--- a/scripts/install-sdk.sh
+++ b/scripts/install-sdk.sh
@@ -102,7 +102,7 @@ installGlobalDeps() {
 NPM=npm
 NODE=node
 
-updateCore || true
+#updateCore || true
 
 installGlobalDeps
 updateAllPackages
-- 
2.1.4

