From ce204e349e4ed51e292ce6b898d86b18aab1db33 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 5 Dec 2017 11:31:14 -0600
Subject: [PATCH 4/4] core dont updateCore we checkout a sha commit and dont
 autorun

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 scripts/install-sdk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/install-sdk.sh b/scripts/install-sdk.sh
index 7acfd55b..ac6685ce 100755
--- a/scripts/install-sdk.sh
+++ b/scripts/install-sdk.sh
@@ -106,7 +106,7 @@ NODE=node
 rm -rf ./build/standalone
 
 # pull the latest version
-updateCore || true
+#updateCore || true
 
 installGlobalDeps
 updateNodeModules
-- 
2.15.0

