From ad059e88f79d153f24a1ee01b043ffdc41e28775 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 25 Apr 2017 21:51:13 -0500
Subject: [PATCH 4/4] core dont updateCore we checkout a sha commit and dont
 autorun

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 scripts/install-sdk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/install-sdk.sh b/scripts/install-sdk.sh
index 7acfd55b..ac6685ce 100755
--- a/scripts/install-sdk.sh
+++ b/scripts/install-sdk.sh
@@ -106,7 +106,7 @@ NODE=node
 rm -rf ./build/standalone
 
 # pull the latest version
-updateCore || true
+#updateCore || true
 
 installGlobalDeps
 updateNodeModules
-- 
2.11.0

