From 80c6f13a8c802dad13b80afb78354c453714b5c5 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 24 Apr 2018 09:49:54 -0500
Subject: [PATCH 4/4] core dont updateCore we checkout a sha commit and dont
 autorun

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 scripts/install-sdk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/install-sdk.sh b/scripts/install-sdk.sh
index 00335604..2b928265 100755
--- a/scripts/install-sdk.sh
+++ b/scripts/install-sdk.sh
@@ -108,7 +108,7 @@ NODE=node
 rm -rf ./build/standalone
 
 # pull the latest version
-updateCore || true
+#updateCore || true
 
 installGlobalDeps
 updateNodeModules
-- 
2.17.0

