From 6dcb2765b78ed1740f83e2f5ba215b9e05aa0f9a Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 12 Jun 2018 09:43:13 -0500
Subject: [PATCH 4/5] core dont updateCore we checkout a sha commit and dont
 autorun

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 scripts/install-sdk.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/install-sdk.sh b/scripts/install-sdk.sh
index 00335604..2b928265 100755
--- a/scripts/install-sdk.sh
+++ b/scripts/install-sdk.sh
@@ -108,7 +108,7 @@ NODE=node
 rm -rf ./build/standalone
 
 # pull the latest version
-updateCore || true
+#updateCore || true
 
 installGlobalDeps
 updateNodeModules
-- 
2.17.1

